apps/rest-api-app/build.gradle.kts

```diff
@@ -1,5 +1,6 @@
 plugins {
     id("bisq.java-library")
+    id("bisq.protobuf")
```

Comment:
Why is that needed? We don't have protobuf files here.

Reply:
Ah saw now you added a protobuf file for AddressDetails.

What is the intention? We use protobuf only or p2p network communication. Here it seems it is just a dto for the response. If so, I would suggest to remove the protobuf part and move the file into report package.

Reply:
Ah, okay. I wasn't aware that Protobufs are primarily for P2P communication. I added them because you mentioned we should avoid using DTOs as much as possible. Since I needed one, and Protobufs seemed like the "state of the art" solution to me as a beginner, I implemented it that way.

But regarding the solution:
You mean I should remove the line id("bisq.protobuf") from build.gradle.kts, which means Protobuf won't be generated, and as a result, AddressDetails would need to be converted to AddressDetailsDto.
Or should Protobuf still be used? If so, I'm not very familiar with the build process: where exactly should I then define the usage of Protobuf(id("bisq.protobuf"))?

Reply:
Yes the id("bisq.protobuf") from gradle and `rest_api.proto` file can be removed as well as the `implements NetworkProto` and the 3 PB related methods in the `AddressDetails` class. The `void verify()` can be kept if you like, but its also mainly for network data to protect against ddos. 
And move that class to `bisq.rest_api.report`.